chemprop/train/cross_validate.py

```diff
@@ -63,6 +64,14 @@ def cross_validate(args: TrainArgs,
     data = get_data(path=args.data_path, args=args, logger=logger, skip_none_targets=True)
     validate_dataset_type(data, dataset_type=args.dataset_type)
     args.features_size = data.features_size()
+    args.atom_features_size = 0
+
+    if args.atom_descriptors == 'descriptor':
+        args.atom_descriptors_size = data.atom_descriptors_size()
+        args.ffn_hidden_size += args.atom_descriptors_size
```

Comment:
Is this necessary? You increase the `first_linear_dim` appropriately in `model.py` but after that I think you can keep the `ffn_hidden_size` the same as before without adding `atom_descriptors_size`.